web/api_test.go

```diff
@@ -29,6 +31,8 @@ func TestApi_ParsePaginatedRequest(t *testing.T) {
 
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
+			t.Parallel()
```

Comment:
Let’s remove the t.Parallel()s from inside the subtests please. We’re already highly parallel and there are some subtle gotchas w subtests that just aren’t worth this addition.